ouroboros-consensus/src/Ouroboros/Consensus/Node/ExitPolicy.hs

```diff
@@ -0,0 +1,31 @@
+module Ouroboros.Consensus.Node.ExitPolicy
+  ( NodeToNodeClientResult (..)
+  , returnPolicy
+  -- Re-exports
```

Comment:
I suspect `stylish-haskell` might object here, unless you write `-- * Re-exports`.

Reply:
it did, but not because of missing ` *` but indentation :D, I'll fix both.